### PR TITLE
Add datadog agent

### PR DESCRIPTION
**Pull request by tauffredou**

### CODE REVIEW

File: datadog/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for Kubernetes
```
Comment (gearnode) [inline datadog/Chart.yaml]: Use the real chart description.

File: datadog/templates/rbac.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
+{{- if .Values.rbac.enabled }}
+
+---
```
Comment (gearnode) [inline datadog/templates/rbac.yaml]: I our cluster `Tiller` have the right to create a `RBAC` ? :astonished:

Comment (tauffredou) [reply datadog/templates/rbac.yaml]: Only in the protected kube-system namespace)

File: datadog/templates/rbac.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+subjects:
+- kind: ServiceAccount
+  name: {{ template "datadog.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+roleRef:
+  kind: ClusterRole
+  name: {{ template "datadog.fullname" . }}
+  apiGroup: rbac.authorization.k8s.io
+
+{{- end }}
```
Comment (gearnode) [inline datadog/templates/rbac.yaml]: Missing `\n`

**Pull request merged**